packages/e2e-tests/helpers/utils.ts

```diff
@@ -179,3 +179,53 @@ export async function resetTestUser(email: string) {
     },
   });
 }
+
+export async function cloneTestRecording(recordingId: string): Promise<string> {
+  const variables = { recordingId, secret: process.env.AUTOMATED_TEST_SECRET };
+
+  var startTime = performance.now();
+  const clonedRecording = await axios({
+    url: config.graphqlUrl,
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY}`,
+    },
+    data: {
+      query: `
+        mutation cloneTestRecording($recordingId: UUID!, $secret: String!) {
+          cloneTestRecording(input: { recordingId: $recordingId, secret: $secret }) {
+            recordingId
+          }
+        }
+      `,
+      variables,
+    },
+  });
+  var endTime = performance.now();
+  const timeInSecs = (endTime - startTime) / 1000;
+  console.log(`Cloning took ${timeInSecs} seconds.`);
+
+  return clonedRecording.data.data.cloneTestRecording.recordingId;
+}
+
+export async function deleteTestRecording(recordingId: string) {
+  const variables = { recordingId, secret: process.env.AUTOMATED_TEST_SECRET };
+
+  return axios({
+    url: config.graphqlUrl,
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY}`,
+    },
+    data: {
+      query: `
+          mutation deleteTestRecording($recordingId: UUID!, $secret: String!) {
```

Comment:
I think we'll also need a backend process that runs periodically and cleans up tests that are older than some threshold (an hour?) because we can't count on JS to always complete.

Reply:
We are keeping track which recording are clones (`cloned_from` field in `recordings` table) but we don't have scheduled deletion implemented yet but as needed we can and execute on these dangling recording.

Reply:
Gotcha! Seems like something we will want to add so it's good we have an easy way to identify these.